spec/controllers/supplejack_api/users_controller_spec.rb

```diff
@@ -11,36 +11,71 @@ module SupplejackApi
   describe UsersController, type: :controller do
     routes { SupplejackApi::Engine.routes }
 
-    let(:user) { FactoryGirl.create(:user, authentication_token: 'abc123', role: 'admin') }
+    context 'admin user' do
+      let(:user) { FactoryGirl.create(:admin_user, authentication_token: 'abc123') }
```

Comment:
Do we include the FactoryGirl syntax methods? If so we can remove all the FactoryGirl calls :D!

Reply:
sure!